cmd/plasmacli/subcmd/keys/list.go

```diff
@@ -24,10 +26,14 @@ var listCmd = &cobra.Command{
 		}
 		defer db.Close()
 
-		fmt.Printf("NAME:\t\tADDRESS:\n")
+		w := new(tabwriter.Writer)
+		w.Init(os.Stdout, 0, 8, 0, '\t', 0)
```

Comment:
Can we add a quick comment saying that this is setting the tab width to 8

Reply:
Done